extensions/wrapper/wrapper-api/src/main/java/de/sovity/edc/ext/wrapper/api/ui/model/ContractTerminationInitiator.java

```diff
@@ -0,0 +1,23 @@
+/*
+ *  Copyright (c) 2024 sovity GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       sovity GmbH - initial API and implementation
+ */
+
+package de.sovity.edc.ext.wrapper.api.ui.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+public enum ContractTerminationInitiator {
+    @Schema(description = "The contract termination was initiated by this EDC.")
+    SELF,
+    @Schema(description = "The contract termination was initiated by the counterparty EDC.")
```

Comment:
this does nothing, here you can prefer java doc, or skip it

Reply:
Removed.